client/COPY.json

```
@@ -374,6 +374,9 @@
   "SEND_CAVC_REMAND_PROCESSED_LETTER_TASK_LABEL": "Send CAVC-Remand-Processed Letter Task",
 
   "CAVC_SEND_TO_TEAM_NON_BLOCKING_DETAIL": "Create a %s assigned to the %s team. This will not block you from completing this task or sending the 90-day letter to the veteran.",
+  "CAVC_SEND_TO_TEAM_BLOCKING_CAVC_DETAIL": "Create a %s task assigned to the %s team. This will block you from sending the 90-day letter to the veteran until the new task is completed.",
+
+  "CAVC_POA_TASK_LABEL": "Verify Power of Attorney",
```

To be consistent with other verbiage, e.g., `CLARIFY_POA_BLOCKING_CAVC` and "Assign POA Clarification task to CAVC team"
```suggestion
  "CAVC_POA_TASK_LABEL": "Clarify Power of Attorney",
```